src/app/customer/error/CreateCustomerAddressError.ts

```diff
@@ -0,0 +1,14 @@
+import { setPrototypeOf, CustomError } from '../../common/error';
+import { getLanguageService } from '../../locale';
+
+export default class CreateCustomerAddressError extends CustomError {
+    constructor(data: Error) {
+        super({
+            name: 'CREATE_CUSTOMER_ADDRESS_FAILED',
+            message: getLanguageService().translate('customer.create_address_error'),
```

Comment:
Generally speaking it is not recommended to have translation string as the message of `Error` instance. This is because if the error bubbles up and get logged by the Sentry logger. The translated error message will be logged. If the error message is translated into a different language (or re-worded in English), it'll create a separate Sentry error report than the ones that are not translated. So it'd make our logs noisy. Another reason is that we probably don't know how to read the message in another language, and it'd take us time to figure out what it actually means.